src/kubectl-object-model.ts

```diff
@@ -79,6 +79,9 @@ export class Kubectl {
         return results;
     }
 
+    public executeCommand(customCommand: string, args?: string) {
+        return args ? this.execute([customCommand, args]) : this.execute([customCommand]);

```

Comment:
Add null check for `customCommand` as well

Reply:
Should I throw an error when no command is given, or should I just return from the function with null?

Reply:
What's the current usecase for this function? Do you want to break it when the command is set to an empty string?

Reply:
The function is currently being used to execute kubectl api-versions command only. I think it makes more sense to throw an error in case of empty command string.

Reply:
Makes sense to throw it then :)